### ISSUE TITLE
Feature Request: Add Ability to Pause/Resume Stream Processing

### ISSUE THREAD

**Issue by valeeum** [moleculerjs/moleculer-channels#48]
It would be nice to be able to pause/resume processing of messages without unsubscribing from the channel.

**Comment by intech** [comment]
@valeeum Do I understand that correctly you want to do this inside the handler? Should it work like in Kafka?

**Comment by valeeum** [comment]
@intech yes, i'd like to do this in the handler.  i unfortunately don't have much experience with kafka but my use case is to be able to pause processing of events during times of system maintenance.

**Comment by icebob** [comment]
Good idea.
